e2e/test/scenarios/dashboard-cards/dashboard-card-resizing.cy.spec.js

```diff
@@ -217,8 +217,27 @@ describe("scenarios > dashboard card resizing", () => {
           cy.findByLabelText("Add questions").click();
         });
 
-        TEST_QUESTIONS.forEach(question => {
-          cy.findByLabelText(question.name).click();
+        /**
+         * Metabase sorts all questions in the sidebar alphabetically.
+         * It makes sense to sort them out here as well in order to avoid
+         * Cypress "jumping" up and down while clicking on them.
+         * It will go in order from top to the bottom, which scrolls the
+         * sidebar naturally. This prevents acting on an element that's not visible.
+         */
+        const sortedCards = TEST_QUESTIONS.sort((a, b) =>
+          a.name.localeCompare(b.name),
+        );
+
+        /**
+         * After each card is added to the dashboard from the sidebar, there is a
+         * request to load the card query. We need to wait for each of those before
+         * we attempt to add a new card. Otherwise the save dashboard might fail
+         * because Cypress is too fast.
+         */
+        cy.intercept("POST", "/api/card/**/query").as("cardQuery");
```

Comment:
This glob matches both `POST /api/card/:id/query` and `POST /api/card/pivot/:id/query`